src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1670,6 +1669,25 @@ private void OnRenderPackageUpdateCompleted(AsyncTask asyncTask)
             if (task.RenderPackages.Any())
             {
                 OnRenderPackagesUpdated(task.RenderPackages);
+
+                OnRequestRenderPackages();
```

Comment:
Raises the event that requests render packages for any manipulators associated with the node. This event is registered with the `DrawManipulator` method in `Manipulator` base class in `DynamoManipulation` project. See PR: https://git.autodesk.com/Dynamo/DynamoPro/pull/178